include/zephyr/bluetooth/audio/has.h

```diff
@@ -95,6 +112,39 @@ struct bt_has_client_cb {
 	 * @param index Active preset index.
 	 */
 	void (*preset_switch)(struct bt_has *has, uint8_t index);
+
+	/**
+	 * @brief Callback function for Hearing Access Service operation status.
+	 *
+	 * The callback called when the control point operation is complete.
+	 * The operations include:
+	 *     @ref bt_has_client_presets_read(),
+	 *
+	 * @param has Pointer to the Hearing Access Service object.
+	 * @param err 0 on success, ATT error or negative errno otherwise.
+	 * @param user_data User data given.
+	 */
+	void (*status)(struct bt_has *has, int err, void *user_data);
+
+	/**
+	 * @brief Callback function for preset list events.
+	 *
+	 * The callback called when the preset record update is notified by the remote server.
+	 *
+	 * @param has Pointer to the Hearing Access Service object.
+	 * @param type Update type.
+	 * @param index Preset index.
+	 * @param value Optional value or NULL.
+	 *		To be able to read it's value must be cast to an appropriate type.
+	 *              - @ref bt_has_preset_record_val when @p type is
+	 *                @ref BT_HAS_PRESET_RECORD_UPDATE or @ref BT_HAS_PRESET_RECORD_READ_RSP.
+	 *              - @ref bt_has_preset_delete_val when @p type is @ref BT_HAS_PRESET_DELETE.
+	 *              The param object as well as it's objects are temporary and must be copied
+	 *              to in order to cache its information.
+	 * @param is_last True if preset list update operation can be considered concluded.
+	 */
+	void (*preset_event)(struct bt_has *has, enum bt_has_preset_event_type type, uint8_t index,
+			     const void *value, bool is_last);
```

Comment:
Yes, I have considered. I decided to have one because, This one callback would be used to handle preset list updates. Moreover `Generic Update` notification and `Read Preset Response` are so similar I wanted to have one, common callback for those